README.md

```diff
@@ -97,138 +97,148 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.73 - August 2023 Update
+### 0.74 - September 2023 Update
 
-In this release, we focused on releasing new features, stability and improvements.
+In this release, we focused on stability and improvements.
 
 **Highlights**
 
- - Keyboard manager now allows Numpad keys to be bound separately from the main keyboard. If using a previous configuration, only the main keyboard (not numpad) keys will be mapped. To restore the previous behavior, manually add separate bindings for numpad keys.
- - New utility: Crop And Lock allows you to crop a current application into a smaller window or just create a thumbnail. Focus the target window and press the shortcut to start cropping.
- - FancyZones code improvements and refactor.
- - Modernized ImageResizer UX.
- - PowerRename advanced counter functionality.
+ - Upgraded to Windows App SDK 1.4.1, increasing stability of WinUI3 utilities. Thanks [@dongle-the-gadget](https://github.com/dongle-the-gadget) for starting the upgrade!
+ - Text Extractor was upgraded to its version 2.0, with a new overlay, table mode and more Quality of Life improvements. Thanks [@TheJoeFin](https://github.com/TheJoeFin)!
+ - Improved FancyZones stability, fixing some layout resets and improving handling of newly created windows on Windows 11.
+ - Fixed many silent crashes that were reported to Watson and the user's event viewer.
+ - Fixes and improvements for PowerToys Run plugins. Thanks [@htcfreek](https://github.com/htcfreek) for the new PowerToys Run plugin settings types! *Attention*: A breaking change is planned for 0.75, in which each plugin has to declare its identifier programmatically so that it can be controlled through GPO. For third-party plugin developers, please check https://github.com/microsoft/PowerToys/pull/27468 for more details.
```

Comment:
```suggestion
 - PowerToys Run plugins now can have additional settings types: text, number, combobox and combinations with checkbox.
```

Reply:
I was thinking this is a bit more hidden, and not very user impacting at this moment, so I wouldn't add it to highlights. Makes sense?

Reply:
Ended up removing from hightlights. @htcfreek , how about editing the line in PowerToys Run section for more details instead?

Reply:
I am sure we have the details ther. Or not?

Reply:
You're right, we do 😄